ComputeClient/Compute.Contracts/Network20/Server.cs

```diff
@@ -1868,28 +1868,40 @@ public string id
     public partial class AdvancedVirtualizationSettingsType
     {
         /// <remarks/>
-        public bool nestedHardwareVirtualization;
+        public bool? nestedHardwareVirtualization;
```

Comment:
Looks good. Does the xsd generator mark these fields  as nullable (using this question mark syntax) or did you have to put it in by hand?

Reply:
I've added by hand because of the decision on implementing the get method to sync the 'specified' property with the real property

Reply:
perfect thought as much - just so i know if i come across this issue again in a few months when dealing the bools. Its good to know 👍